src/index.ts

```diff
@@ -111,17 +118,70 @@ module.exports = (app: Probot) => {
                   await git.checkout(prButBetter.base.ref);
                   await git.checkout(prButBetter.head.ref);
 
+                  app.log.info(`Checked out for pr: #${pr.id}`);
+
                   // Merge base to head
-                  const { result } = await git.mergeFromTo(
-                    prButBetter.base.ref,
-                    prButBetter.head.ref,
-                  );
+                  let result = 'failed';
+                  try {
+                    await git.raw([
+                      'merge',
+                      prButBetter.base.ref,
+                      '-m',
+                      `[ci skip] Merged in branch '${prButBetter.base.ref}' to fix mergability`,
+                    ]);
+                    result = 'success';
+                  } catch (err) {
+                    // Failed to merge
```

Comment:
can we log if it fails to merge

Reply:
We do elsewhere because of the `return false;`